src/Kafka/CreateKafkaInstance/CreateKafkaInstance.tsx

```diff
@@ -119,6 +119,15 @@ export const CreateKafkaInstance: FunctionComponent<
     [create]
   );
 
+  if (!isTrial) {
+    ingresEgress = 100;
```

Comment:
For ingressEgress there are two values in doc 100 and 50.

Reply:
This is for standard instance

Reply:
You're correct. The updates we made for [the new UI](https://redhat-developer.github.io/app-services-ui-components/?path=/story/kafka-create-kafka-instance-with-sizes-components-instance-info--quota-instance) split these out into separate values. I'm not sure what our options are for splitting them out for this PR. Maybe this is something that @riccardo-forina can provide a suggestion for.